contributor-docs/operations/tests/rfc2119-keyword-test-guide.md

```diff
@@ -0,0 +1,157 @@
+---
+title: SCS RFC2119 Keyword Test Guide
+type:
+status: Draft
+track: Global
+---
+
+## Introduction
+
+The process of test creation and standards definition in Sovereign Cloud Stack (SCS)
+involves developing and validating tests that align with the specific requirements and
+standards set by SCS. This entails a comprehensive understanding of the SCS framework,
+which is built on open-source components and aims to provide interoperable and sovereign
+cloud services. In creating tests for SCS, it's essential to consider the various aspects
+of cloud technology, including infrastructure, services, and the integration of these
+components. The tests should be designed to ensure that the SCS standards are met, which
+includes adhering to specified requirement levels as indicated by [RFC2119 keywords](https://datatracker.ietf.org/doc/html/rfc2119).
+
+## 1. Understanding SCS Standards and RFC2119 Keywords
+
+SCS standards use RFC2119 keywords like:
+
+- **MUST** (same as **REQUIRED**, **SHALL**),
+- **MUST NOT** (same as **SHALL NOT**),
+- **SHOULD** (same as **RECOMMENDED**),
+- **SHOULD NOT** (same as **NOT RECOMMENDED**),
+- **MAY** (same as **OPTIONAL**)
+
+to define requirements for SCS-compatible IaaS and KaaS resources. These standards are crucial for ensuring interoperability and sovereignty in cloud services, as they are built on open-source components such as Kubernetes.
+
+To ensure readability and comprehensibility, only the main keywords **MUST**, **MUST NOT**, **SHOULD**, **SHOULD NOT** and **MAY** are referred to below.
+
+## 2. Interpreting RFC2119 Keywords in the Context of SCS
+
+- **MUST and MUST NOT**: These are binary and straightforward to test. Compliance or
+  non-compliance directly affects the standard meeting. For visualization, fulfilled
```

Comment:
```suggestion
  non-compliance directly affects whether the standard is satisfied. For visualization, fulfilled
```

(To me, *standard meeting* sounds like something were people meet.)

Reply:
changed